react-components/tsconfig.json

```diff
@@ -5,19 +5,14 @@
     "strict": true,
     "skipLibCheck": false,
     "jsx": "react-jsx",
-    "module": "ESNext",
+    "module": "ES2020",
     "declaration": true,
     "declarationDir": "dist",
     "declarationMap": true,
     "sourceMap": true,
     "outDir": "dist",
     "moduleResolution": "bundler",
-    "allowSyntheticDefaultImports": true,
+    "allowSyntheticDefaultImports": true
   },
-  "include": [
-    "src"
-  ],
-  "exclude": [
-    "**/*.stories.ts"
-  ]
-}
\ No newline at end of file
+  "include": ["src/*", "stories/*"]
+}
```

Comment:
Is this right? It seems that `stories` is moved from `exclude` to `include`

Reply:
ts gets mad at me if I don't add this :(